### PR TITLE
fix: `docai_utilities.py` to return `Optional`

### PR DESCRIPTION
**Pull request by holtskinner**
- Should resolve customer reported issue in support case #47169701 relating to duplicate/inaccurate elements in hOCR output
- Followup to:
  - #161
  - #169

### CODE REVIEW

File: google/cloud/documentai_toolbox/utilities/docai_utilities.py [hunk 1]
```diff
@@ -37,8 +37,8 @@ def get_bounding_box(
             Bounding box coordinates in order (top, left, bottom, right).
             Returns `0, 0, 0, 0` if `bounding_poly.normalized_vertices` is empty.
     """
-    if not bounding_poly.normalized_vertices:
-        return 0, 0, 0, 0
+    if not bounding_poly or not bounding_poly.normalized_vertices:
```
Comment (dizcology) [inline google/cloud/documentai_toolbox/utilities/docai_utilities.py:40]: Update the docstring for this changed behavior.

File: google/cloud/documentai_toolbox/wrappers/entity.py [hunk 2]
```diff
@@ -95,9 +95,11 @@ def crop_image(
         if not documentai_page.image:
             raise ValueError("Document does not contain images.")
 
-        top, left, bottom, right = docai_utilities.get_bounding_box(
+        bbox = docai_utilities.get_bounding_box(
             bounding_poly=self.documentai_object.page_anchor.page_refs[0].bounding_poly,
             page_dimension=documentai_page.dimension,
         )
+        if not bbox:
```
Comment (dizcology) [inline google/cloud/documentai_toolbox/wrappers/entity.py:102]: In this case let's use `if bbox is None`.  Also perhaps move this check to immediately after we get `bbox`.

File: tests/unit/test_document.py [hunk 3]
```diff
@@ -648,6 +656,29 @@ def test_export_images(get_bytes_images_mock):
...
 
 
+def test_export_images_empty_bounding_box(get_bytes_images_mock):
+    output_path = "resources/output/"
+
+    doc = document.Document.from_gcs(
+        gcs_bucket_name="test-directory", gcs_prefix="documentai/output/123456789/0"
+    )
+
+    del (
```
Comment (dizcology) [inline tests/unit/test_document.py:666]: Perhaps put these `doc = ...; del(...` lines of code into a factory function, say, `def make_document_without_bbox`, and each test that needs a document of that shape can call the function to get a copy of it.

**Review by dizcology — CHANGES_REQUESTED** [review]

**Comment by holtskinner** [comment]
> If possible, please also give some explanations about the duplicate/inaccurate elements in hOCR output in the PR description.

I will once I get that information

File: google/cloud/documentai_toolbox/utilities/docai_utilities.py (hunk 1 above)
Comment (holtskinner) [reply google/cloud/documentai_toolbox/utilities/docai_utilities.py:40]: Missed this.

File: google/cloud/documentai_toolbox/wrappers/entity.py (hunk 2 above)
Comment (holtskinner) [reply google/cloud/documentai_toolbox/wrappers/entity.py:102]: Done, although I'm not sure of your second comment. This is already right after we get bbox.

**Comment by holtskinner** [comment]
Note: In the customers code, they use this whenever the document is blank. Not sure if this is a standard structure for blank hOCR documents, but could be good to look into.

```py
    def _build_empty_hocr(self):
        return '''<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="unknown" lang="unknown">
<head>
<title>hocr</title>
<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
<meta name="ocr-system" content="Document AI OCR" />
<meta name="ocr-langs" content="unknown" />
<meta name="ocr-number-of-pages" content="1" />
<meta name="ocr-capabilities" content="ocr_page ocr_carea ocr_par ocr_line ocrx_word" />
</head>
<body>
<div class='ocr_page' lang='unknown' title='bbox 0 0 0 0'>
<span class='ocr_carea' id='block_1_0' title='bbox 0 0 0 0'>
<span class='ocr_par' id='par_1_0_0' title='bbox 0 0 0 0'>
<span class='ocr_line' id='line_1_0_0_0' title='bbox 0 0 0 0'></span>
<span class='ocrx_word' id='word_1_0_0_0_0' title='bbox 0 0 0 0'></span>
</span>
</span>
</div>
</body>
</html>'''
```

**Review by dizcology — APPROVED** [review]

**Pull request merged**